index.js

```diff
@@ -50,29 +50,31 @@ module.exports = function(content){
 		if(debug){
 			console.log(cmd);
 		}
-		let {stdout, stderr} = await exec(cmd);
-		if(debug){
-			console.log('stdout:', stdout);
-			console.log('stderr:', stderr);
-		}
-		
-		if(stderr){
-			self.emitError(stderr);
-			callback(stderr);
-		}
-		else{
-			// Split out last line which contains list of included files and add them as Webpack dependencies
-			const includedSeparator = stdout.lastIndexOf('\n');
-			const dependencies = stdout.slice(includedSeparator);
-			stdout = stdout.slice(0, includedSeparator);
+		try{
+			let {stdout, stderr} = await exec(cmd);
+			if(debug){
+				console.log('stdout:', stdout);
+				console.log('stderr:', stderr);
+			}
+
+			if(stderr){
```

Comment:
I'm not sure when this gets hit as it seems to throw a promise rejection instead. Maybe this could be consolidated with the catch logic somehow or removed.